### PR TITLE
First draft of how we hire doc

### PR DESCRIPTION
**Pull request by davisshaver**

### CODE REVIEW

**Comment by davisshaver** [comment]
First draft could use some eyes. Introduces the `pre-app` and `app` as the new trial project pitch form and trial project names.

File: team-culture/how-we-hire.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # How we hire
+Fusion tech hires differently than most media companies.
 
-Coming soon: documentation of our hiring process. To include:
+We'll chat with anyone who wants to learn about our work. We publish about our projects and processes often, and we love talking to others who are excited about this stuff. Not an interview, a conversation.
```
Comment (danielbachhuber) [inline team-culture/how-we-hire.md]: Should we have a subhead for "Conversations"

File: team-culture/how-we-hire.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
 
-Coming soon: documentation of our hiring process. To include:
+We'll chat with anyone who wants to learn about our work. We publish about our projects and processes often, and we love talking to others who are excited about this stuff. Not an interview, a conversation. 
 
-* Interviews
-* Our "applications"
+As part of this process, you'll also get access to our pre-app form. The pre-app form can be submitted anytime after your first chat with a Fusion team member. Sometimes candidates will talk to more than one team member before submitting.
+
+The pre-app form pitches a mini-project that will let our team work with you for a brief period of time on a topic of shared interest. Maybe it's an open source project that needs some help finishing a release or feature, a pain point we mentioned in a conversation, a tool you've always wanted to build that sounds friggin' awesome (our words, not yours).
+
```
Comment (danielbachhuber) [inline team-culture/how-we-hire.md]: Worth mentioning we can help suggest ideas?

**Comment by danielbachhuber** [comment]
Good start. Can you include some mention about our interview / conversation process, and we can merge and iterate from there?

File: team-culture/how-we-hire.md (hunk 1 above)
Comment (davisshaver) [inline team-culture/how-we-hire.md]: @danielbachhuber torn on that but we can try. Thought it was pretty short though

**Comment by davisshaver** [comment]
@danielbachhuber sorry not sure I follow the comment – where am I mentioning the process?

**Comment by danielbachhuber** [comment]
> where am I mentioning the process?

At the beginning?

**Comment by davisshaver** [comment]
@danielbachhuber gotcha. like that?

**Comment by danielbachhuber** [comment]
Yep, pretty good start. We can evolve from here.

**Pull request merged**